digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala

```diff
@@ -38,29 +57,66 @@ class SearchApi(roadAddressService: RoadAddressService) extends  ScalatraServlet
     }
   }
 
-  get("/road_numbers?") {
+  private val getRoadNumbers: SwaggerSupportSyntax.OperationBuilder =
+    (apiOperation[Seq[Long]]("getRoadNumbers")
+      tags "SearchAPI"
+      summary "getRoadNumbers."
+      )
+
+  get("/road_numbers?", operation(getRoadNumbers)) {
     time(logger, "GET request for /road_numbers?") {
       roadAddressService.getRoadNumbers
     }
   }
 
-  get("/road_address/:road/?") {
+  private val getRoadAddressWithRoadNumber: SwaggerSupportSyntax.OperationBuilder = (
+    apiOperation[List[Map[String, Any]]]("getRoadAddressWithRoadNumber")
+      .parameters(
+        pathParam[Long]("road").description("roadNumber")
```

Comment:
Description could be more human friendly.

Reply:
I do not know application very well. If I make more human friendly description I create it by guessing. I tested each api and they work and live result is documentation itself. This is what i could do with original 1 storypoint limitation.

Reply:
fixed